src/sentry/static/sentry/app/views/performance/landing.tsx

```diff
@@ -265,44 +261,29 @@ class PerformanceLanding extends React.Component<Props, State> {
   }
 
   renderHeaderButtons() {
+    const {organization} = this.props;
+    const views: FilterViews[] = [FilterViews.ALL_TRANSACTIONS];
+    if (!organization.features.includes('key-transactions')) {
```

Comment:
```suggestion
    if (organization.features.includes('key-transactions')) {
```

I think this is a typo.

Reply:
Ah. Nevermind.